common/transforms/und-Ethi-t-und-latn-m0-beta_metsehaf-geminate.xml

```diff
@@ -19,7 +19,7 @@
 ########################################################################
 #
 :: Amharic-Amharic/Geminate ; 
-:: Ethiopic-Latin/BetaMetsehaf ;
+:: Ethiopic-Latin/Beta_Metsehaf ;
```

Comment:
This is needed for consistency with the related file und-Ethi-t-und-latn-m0-beta_metsehaf.xml